File: js/test/kata8/evenorodd.test.js
const eod = require('../../kata8/evenorodd.js');

test('2 is even', () => {
    expect(eod(2)).toBe('Even');
})

test('0 is even', () => {
    expect(eod(0)).toBe('Even');
})

test('7 is odd', () => {
    expect(eod(7)).toBe('Odd');
})

test('1 is odd', () => {
    expect(eod(1)).toBe('Odd');
})

File: java/test/codeWars/kata8/EvenOrOddTest.java
package codeWars.kata8;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class EvenOrOddTest {
	
	@Test
	public void evenOrOdd() {
		assertEquals(EvenOrOdd.evenOrOdd(6), "Even");
		assertEquals(EvenOrOdd.evenOrOdd(7), "Odd");
	}
}